### DISCUSSION TITLE
TuringCourses/major/theory_of_computation/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#106] [General]
# TuringCourses/major/theory_of_computation/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/theory_of_computation/

**Comment by MTDickens** [comment]
今年秋冬只有毛宇尘教这门“理论计算机科学导引”，金小刚反而去教“计算理论”了。

**Comment by ken0yuan** [comment]
24-25平时作业有占比，期末可以覆盖小测，这门课的学分和内容量不符，是大三上的一门硬课，图灵机后面的内容尤其硬。
